development/auto-changelog.js

```diff
@@ -96,8 +96,22 @@ async function main() {
     const firstReleaseHeaderIndex = changelogLines.findIndex((line) =>
       line.match(/## \[\d+\.\d+\.\d+\]/u),
     );
+    const [, previousVersion] = changelogLines[firstReleaseHeaderIndex].match(
+      /## \[(\d+\.\d+\.\d+)\]/u,
+    );
     changelogLines.splice(firstReleaseHeaderIndex, 0, versionHeader, '');
     releaseHeaderIndex = firstReleaseHeaderIndex;
+
+    // Update release links
```

Comment:
nit: "links" is a confusing reference. It would be better if the comment stated that it's rewriting the `Unreleased` link and inserting a link for new version.

Reply:
To be more specific, it's a link reference definition: https://github.github.com/gfm/#link-reference-definitions
I'll update the comment to better explain this.

Reply:
Addressed here: bad85c4fd6c825faffca9b2f89f2e9d928bfe6af